backend/tests/integration/editor/test_editor_views.py

```diff
@@ -510,64 +510,95 @@ def test_refextract_url(inspire_app):
 
 def test_file_upload(inspire_app, s3, datadir, override_config):
     current_s3_instance.client.create_bucket(Bucket="inspire-editor")
+    hidden_collection = "HEP Hidden"
     user = create_user(role=Roles.cataloger.value)
-
+    record = create_record("lit", data={"_collections": [hidden_collection]})
```

Comment:
I would create a separate test with a user that is not curator, we have to keep these tests as is